bin/build-cloud

```diff
@@ -0,0 +1,35 @@
+ #! /usr/bin/env bash
+
+ # DOC: Build civiform/cloud.
+
+IMAGE="cloud"
+LOCATION="."
+DOCKERFILE="cloud/cloud.Dockerfile"
+
+BUILD_ARGS=(-f "${DOCKERFILE}"
+  -t "civiform/${IMAGE}:latest"
+  --cache-from "civiform/${IMAGE}"
+  --build-arg BUILDKIT_INLINE_CACHE=1
+  "${LOCATION}")
+
+PLATFORM_ARG=()
+if [[ "${PLATFORM}" ]]; then
+  PLATFORM_ARG=(--platform "${PLATFORM}")
+fi
+
+# Build the multi-platform image
+echo "start ${IMAGE} build"
+docker buildx build "${PLATFORM_ARG[@]}" "${BUILD_ARGS[@]}"
+
+# Load the image from the cache
+echo "load ${IMAGE} build"
+docker buildx build --load "${BUILD_ARGS[@]}"
+
+if [[ "${PUSH_IMAGE}" ]]; then
+  docker::do_dockerhub_login
+  # Push the image from the cache to dockerhub
+  echo "push ${IMAGE} build"
+  docker buildx build --push "${PLATFORM_ARG[@]}" "${BUILD_ARGS[@]}"
+fi
+
+docker tag "civiform/${IMAGE}:latest" "${IMAGE}:latest"
```

Comment:
If we do use something generic (i.e. cloud) as the name, we need to use the name `civiform-cloud` as the local name, so it doesn't conflict with any public packages

Reply:
renamed. if you have better suggestions - let me know, naming is hard.